### ISSUE TITLE
Unable to locate package apktool

### ISSUE THREAD

**Issue by rocky431** [termux/termux-packages#6806]
When i using metasploit and bind a payload with another apk 
Then the error is apktool not found
Please help me

**Comment by agnostic-apollo** [comment]
Check unofficial https://github.com/Lexiie/Termux-Apktool

**Comment by rocky431** [comment]
After that??

**Comment by agnostic-apollo** [comment]
Install instructions are there and we don't provide support for metasploit. Check `Where are Metasploit and Hydra packages`

https://wiki.termux.com/wiki/FAQ

**Comment by rocky431** [comment]
I don't understand

Please help me to install package apktool

**Comment by WMCB-Tech** [comment]
@rocky431 

As @agnostic-apollo stated, you need to install apktool from this unofficial source 

> Check unofficial https://github.com/Lexiie/Termux-Apktool

It contains deb package inside so you get the deb package in that repository then install it via `dpkg`, instructions stated there

**Comment by ChiefMikeK** [comment]
Maybe rephrase the FAQ? 
- As for others... ~~lets they~~ take a <s>mission impossible</s> challenge of installing package from source!
- As for others... let them take a <s>mission impossible</s> challenge of installing package from source!
https://wiki.termux.com/wiki/FAQ#Where_are_Metasploit_and_Hydra_packages

**Comment by Hax4us** [comment]
@rocky431 you can look https://github.com/hax4us/Apkmod

**Comment by rocky431** [comment]
how to add keytool in termux

please give me codes
